chapters/en/slides/module1/module1_03.md

```diff
@@ -1,608 +1,210 @@
-
-Here, we *encode* the column `Miles_per_Gallon` as the x-axis. For
-Pandas data frames, Altair automatically determines an appropriate data
-type for the mapped column, which in this case is quantitative (or
-numerical) and shows the numbers under the axis.
-
-You can see that there are several short black lines spread out evenly
-on the x-axis. These are called axis ticks and help us see where the
-values of this dataframe column lie along the axis.
-
-The faint gray lines are called grid lines and extend the locations of
-the axis ticks so that it is easy to compare their position to the
-points.
-
-This is particularly useful when the points might be further away from
-the axis ticks, such as in the next slide.
-
----
-
-## Mapping a dataframe column to the y-scale
-
-``` python
-alt.Chart(cars).mark_point().encode(
-    x='Weight_in_lbs',
-    y='Miles_per_Gallon')
-```
-
-<iframe src="/module1/charts/03/unnamed-chunk-6.html" width="100%" height="420px" style="border:none;">
-</iframe>
-
-Notes: By spreading out the data along both the x and y-axis, we can
-answer our initial question about the relationship between car weight
-and gas mileage. as it appears that the heavier cars are the ones that
-have the poorest mileage.
-
-Although we might have expected this to be the case, visualizing all the
-data points also provides information on the nature of the relationship
-between weight and mileage.
-
-It appears that the x-y points don’t simply follow a straight line, but
-rather a curved line that where the mileage drop quickly when moving
-away from the lightest cars, but then decreases more slowly throughout
-the remainder of the data.
-
-This rich, easily interpretable display of information is one of the
-main advantages of visualizing data and later in the course, we will
-talk more about the different type of relationships, such as linear,
-exponential, etc.
-
----
-
-## Mapping a numerical dataframe column to the colour scale
-
-``` python
-alt.Chart(cars).mark_point().encode(
-    x='Weight_in_lbs',
-    y='Miles_per_Gallon',
-    color='Horsepower')
-```
-
-<iframe src="/module1/charts/03/unnamed-chunk-7.html" width="100%" height="420px" style="border:none;">
-</iframe>
+This slide outlines the high level parts
+that you want to include in an effective question.
+In essence,
+you want to make your question as easy to understand as possible
+and your specific problem as easy to reproduce as possible.
+
+If you just include a screenshot and title your question "Help",
+the person helping you has to spend time trying to figure out
+what you want help with instead of helping with the actual problem.
+
+To avoid this,
+include a succinct and clear
+description of your problem
+together with the minimal code needed to reproduce it.
+Altogether,
+this is often called a "Minimal Reproducible Example" (or MRE for short),
+which we will learn more about on the next page.
+
+---
+
+## An effective question with a Minimal Reproducible Example
```

Comment:
I think this slide has too much text. If it is not possible to reduce it or change it for an image I would highlight some text.

Reply:
I agree it is quite verbose... I made the key part bold to highlight them